pages/library/_viewId.vue

```diff
@@ -49,6 +49,9 @@ export default Vue.extend({
         });
 
         this.items = itemsResponse.data.Items || [];
+        for (let i = 0; i < 11; ++i) {
+          this.items.push(this.items[0]);
+        }
```

Comment:
?

Reply:
Oops my bad, I only got one item in my library so I practiced some multiplications

Reply:
Done